rasa_nlu/evaluate.py

```diff
@@ -436,6 +436,15 @@ def get_entity_extractors(interpreter):
     return extractors - entity_processors
 
 
+def get_intent_classifier(interpreter):
```

Comment:
This is never used to actually get the classifiers but only to check if one is present. I'd suggest to call this something like `is_intent_classifier_present` and directly return the bool here